code/modules/atmospherics/machinery/components/binary_devices/circulator.dm

```diff
@@ -72,39 +72,36 @@
 	update_icon()
 
 /obj/machinery/atmospherics/binary/circulator/proc/get_inlet_air()
-	if(side_inverted==0)
-		return air2
-	else
+	if(side_inverted)
```

Comment:
The logic for *all* of these `if(side_inverted)` checks are now backwards. Before they checked if it was equal to 0, which should be converted to `if(!side_inverted)`.

Reply:
Woops, I didn't notice that change. All good then.